packages/region-config-resolver/src/regionConfig/getRealRegion.ts

```diff
@@ -0,0 +1,11 @@
+import { isFipsRegion } from "./isFipsRegion";
+
+/**
+ * @internal
+ */
+export const getRealRegion = (region: string) =>
```

Comment:
Out of curiosity, where did the name for this function come from?

I'm wondering if there is a more descriptive name for this.

Reply:
A lot of files from this PR are copied directly from `@smithy/config-resolver`. The names should be kept the same for now.